cms/src/builder.rs

```diff
@@ -99,7 +109,6 @@ pub struct SignerInfoBuilder<'s, S> {
 impl<'s, S> SignerInfoBuilder<'s, S>
```

Comment:
Perhaps this should follow the revised approach in #1161? Though that would require the corresponding changes to the `Builder` trait I suppose.

Reply:
I also think, we should do follow the #1161 `Builder`. But this PR wouldn't work until #1161 is merged. So, it's just a timing problem. It's also possible to merge this PR first and update the CMS crate in #1161, before that is merged.

Reply:
If you're okay with updating `cms` after #1161 is merged, we can go ahead and merge this as-is.

Reply:
I'm okay with updating `cms` after #1161 was merged. But end of this week I plan to be on holidays for about 4 weeks, so this should be taken into account.

Reply:
I'm out for parental leave until the first week of september too, I have some spare time, but kid has some amount of priority here ^^. All that to say, no idea if #1161 will move before you come back.
On top of that, we also need to plan/sequence the next set of breaking changes (If I could throw #1117 in the next train too, that'd be cool).